enterprise-ws-spec/src/main/asciidoc/DeploymentDescriptors.adoc

```diff
@@ -225,38 +216,39 @@ the following information:
 reference that is used in the client source code. It is recommended, but
 not required that the name begin with service/.
 * *Service type:* The service-interface element defines the fully
-qualified name of the JAX-RPC or JAX-WS Service Interface/Class class
-returned by the JNDI lookup.
+qualified name of the Jakarta XML Web Services Service Interface/Class 
+class returned by the JNDI lookup.
 * *Service Reference type:* The service-ref-type element declares the
 type of the service-ref element that is injected or returned when a JNDI
 lookup is done. This must be either a fully qualified name of Service
 class or the fully qualified name of Service endpoint interface class.
-This is an optional element and used only with JAX-WS.
+This is an optional element and used only with Jakarta XML Web Services.
```

Comment:
there is nothing else than `Jakarta XML Web Services.`, so this sentence can be as simple as `This is an optional element.`

Reply:
ok will update as suggested.